app-react/src/main/kotlin/com/ustadmobile/view/ClazzLogEditAttendanceScreen.kt

```diff
@@ -0,0 +1,279 @@
+                    person = Person().apply {
+                        firstNames = "Student Name"
+                    }
+                }
+            ),
+            clazzLogsList = listOf(
+                ClazzLog().apply {
+                    logDate = 1671629979000
+                },
+                ClazzLog().apply {
+                    logDate = 1671975579000
+                },
+                ClazzLog().apply {
+                    logDate = 1671975579000
+                }
+            )
+        )
+    }
+}
+
+private val ClazzLogEditAttendanceScreenComponent2 = FC<ClazzLogEditAttendanceScreenProps> { props ->
+
+    val strings = useStringsXml()
+
+    Container {
+        maxWidth = "lg"
+
+        Stack {
+            direction = responsive(StackDirection.column)
+            spacing = responsive(10.px)
+
+            PagerView{
+                timeZone = props.uiState.timeZone
+                clazzLog = props.uiState.clazzLogsList
+                onClickPreviousClazzLog = props.onClickPreviousClazzLog
+                onClickNextClazzLog = props.onClickNextClazzLog
+            }
+
+            List {
+
+                ListItem {
+                    ListItemButton {
+                        onClick = {props
+                            .onClickMarkAllPresent(ClazzLogAttendanceRecord.STATUS_ATTENDED)
+                        }
+
+                        ListItemIcon {
+                            + LibraryAddCheckOutlined.create()
+                        }
+
+                        ListItemText {
+                            primary = ReactNode(strings[MessageID.mark_all_absent])
+                        }
+                    }
+                }
+
+                ListItem {
+                    ListItemButton {
+                        onClick = {props
+                            .onClickMarkAllAbsent(ClazzLogAttendanceRecord.STATUS_ABSENT)
+                        }
+
+                        ListItemIcon {
+                            + CheckBoxOutlined.create()
+                        }
+
+                        ListItemText {
+                            primary = ReactNode(strings[MessageID.mark_all_absent])
+                        }
+                    }
+                }
+
+                props.uiState.clazzLogAttendanceRecordList.forEach { clazzLogAttendance ->
+
+                    ClazzLogItemView {
+                        clazzLog = clazzLogAttendance
+                        onClazzLogAttendanceChanged = props.onClazzLogAttendanceChanged
+                        fieldsEnabled = props.uiState.fieldsEnabled
+                    }
+
+                }
+            }
+        }
+    }
+}
+
+
+
+external interface PagerViewProps : Props {
+
+    var timeZone: String
+
+    var onClickPreviousClazzLog: () -> Unit
+
+    var onClickNextClazzLog: () -> Unit
+
+    var clazzLog: List<ClazzLog>
+
+}
+
+private val PagerView = FC<PagerViewProps> { props ->
+
+    val dateTime = useFormattedDateAndTime(
```

Comment:
This is hardcoded to  use ClazzLog 0. It needs to use a state variable inside. Clicking needs to trigger the onChangeClazzLog function (so it uses the same function as Android).